internal/otelcollector/config/processors.go

```diff
@@ -57,7 +57,7 @@ type AttributeAction struct {
 }
 
 type TransformProcessorStatements struct {
-	Context    string   `yaml:"context"`
+	Context    string   `yaml:"context,omitempty"`
```

Comment:
Can't we delete if fully? for what is it still used?

Reply:
The attribute removed